xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/LoginIT.java

```diff
@@ -155,10 +154,15 @@ void redirectBackAfterLogin(TestUtils setup, XWikiWebDriver driver, TestReferenc
     @Order(4)
     void correctUrlIsAccessedAfterLogin(TestUtils setup, TestReference testReference)
     {
+        // Logout, then login again to refresh the csrf token required to perform the following operations on
+        // testReference.
+        setup.gotoPage(setup.getURLToLogout());
```

Comment:
From what I see the previous test is actually removing cookies: that would explain the issue here. Have you checked if you reproduce the problem when the previous test is disabled? It might be something to fix in the previous test.